lib/lookup.json

```diff
@@ -505,11 +378,6 @@
     "maintainers": "tj",
     "skip": ">=11"
   },
-  "tape": {
-    "head": true,
-    "prefix": "v",
-    "maintainers": "substack"
-  },
```

Comment:
```suggestion
  "tape": {
    "head": true,
    "prefix": "v",
    "skip": "win32",
    "scripts": ["tests-only"],
    "maintainers": "ljharb"
  },

Reply:
Well, this is kind of annoying. CITGM is supposed to skip `tape` for `"win32"` but it's clearly running (and failing) on win-vs2019: https://ci.nodejs.org/view/Node.js-citgm/job/citgm-smoker/3214/nodes=win-vs2019/testReport/junit/(root)/citgm/tape_v5_6_6/.  I guess the skip logic needs some troubleshooting.

Reply:
i just pushed up a commit that may fix that particular thing, fwiw, but if not then it'd be great to find a way to skip it.